ingestor/src/test/scala/weco/concepts/ingestor/stages/ScrollTest.scala

```diff
@@ -0,0 +1,46 @@
+package weco.concepts.ingestor.stages
+
+import akka.actor.ActorSystem
+import akka.stream.scaladsl.Source
+import akka.stream.testkit.scaladsl.TestSink
+import akka.util.ByteString
+import org.scalatest.funspec.AnyFunSpec
+import org.scalatest.matchers.should.Matchers
+
+import java.io.ByteArrayOutputStream
+import java.util.zip.GZIPOutputStream
+
+class ScrollTest extends AnyFunSpec with Matchers {
+  it("decompresses a gzipped stream and splits it into lines") {
+    implicit val actorSystem: ActorSystem = ActorSystem("test")
+    val lines = Seq(
+      "Freude, schöner Götterfunken",
+      "Tochter aus Elysium,",
```

Comment:
Always love subtle political statements hidden in tests

Reply:
Well it was this or something from Das Kapital